packages/full-stack-tests/src/testing/RulesetTesting.test.ts

```diff
@@ -16,109 +16,136 @@ const iModelPath = "assets/datasets/Properties_60InstancesWithUrl2.ibim";
 
 const MY_HIERARCHY_RULESET: Ruleset = {
   id: "my-test-hierarchy",
-  rules: [{
-    ruleType: RuleTypes.RootNodes,
-    autoExpand: true,
-    specifications: [{
-      specType: ChildNodeSpecificationTypes.InstanceNodesOfSpecificClasses,
-      classes: [{
-        schemaName: "BisCore",
-        classNames: ["Subject"],
-      }],
-      instanceFilter: "this.Parent = NULL",
-      arePolymorphic: false,
-      groupByClass: false,
-      groupByLabel: false,
-    }],
-  }, {
-    ruleType: RuleTypes.ChildNodes,
-    condition: "ParentNode.IsOfClass(\"Subject\", \"BisCore\")",
-    onlyIfNotHandled: true,
-    specifications: [{
-      specType: ChildNodeSpecificationTypes.RelatedInstanceNodes,
-      relationshipPaths: [{
-        relationship: {
-          schemaName: "BisCore",
-          className: "SubjectOwnsSubjects",
+  rules: [
+    {
+      ruleType: RuleTypes.RootNodes,
+      autoExpand: true,
+      specifications: [
+        {
+          specType: ChildNodeSpecificationTypes.InstanceNodesOfSpecificClasses,
+          classes: [
+            {
+              schemaName: "BisCore",
+              classNames: ["Subject"],
+            },
+          ],
+          instanceFilter: "this.Parent = NULL",
+          arePolymorphic: false,
+          groupByClass: false,
+          groupByLabel: false,
         },
-        direction: RelationshipDirection.Forward,
-      }],
-      groupByClass: false,
-      groupByLabel: false,
-    }, {
-      specType: ChildNodeSpecificationTypes.InstanceNodesOfSpecificClasses,
-      classes: {
-        schemaName: "BisCore",
-        classNames: ["Model"],
-      },
-      arePolymorphic: true,
-      relatedInstances: [{
-        relationshipPath: {
-          relationship: {
-            schemaName: "BisCore",
-            className: "ModelModelsElement",
-          },
-          direction: RelationshipDirection.Forward,
-          targetClass: {
+      ],
+    },
+    {
+      ruleType: RuleTypes.ChildNodes,
+      condition: 'ParentNode.IsOfClass("Subject", "BisCore")',
```

Comment:
I'd like to avoid single quotes for strings. Assuming `prettier` only changes double quotes to single ones to avoid escaping double quotes, maybe we could replace single quotes with <code>`</code>?

Reply:
Changed single quotes to backticks.